src/api/index.js

```diff
@@ -0,0 +1,19 @@
+import postsFromServer from './posts.json';
+import commentsFromServer from './comments.json';
+import usersFromServer from './users.json';
+
+export function getUser(userId) {
+  return usersFromServer.find(user => user.id === userId);
```

Comment:
here I would add additional check `|| null`, so if it doesn't find any user then we will return `null`